docs/fieldnorm/soups.md

```diff
@@ -0,0 +1,12 @@
+# SOUPS - Software of Unknown Provenance
+
+Software of unknown provenance (SOUPS) is a software item that is already developed, generally available and not developed for the purpose of being incorporated into the bioinformatic workflow, or a software item for which adequate software documentation is not available. A procedure should be documented about how and where SOUPS are registered.
+
+For each SOUP the following items should be documented:
```

Comment:
I would prefer: "For each SOUP records of the following items should be retained"

I prefer Records over documented as it gives a bit more freedom. For example, you could build into your software that it writes all SOUPS used and there versions into a metainfo file which is shared alongside the results for each sample. This would automatically tick of some of the items.